### ISSUE TITLE
Show some default lines on the homescreen when user enters

### ISSUE THREAD

**Issue by hathix** [trynmaps/metrics-mvp#189]
Right now it's not clear what the map does or how it works. We should show some lines by default (perhaps choose city hall as the start point) so the user knows that they can click and see a spider diagram. Or we can  ask for their location straight away.

**Comment by exxonvaldez** [comment]
I like the idea of starting with City Hall selected, this what Uber Movement does.  I'm thinking we could have two route tables on the right.   If the map has a selected pin, we put up the filtered table titled "Routes Near Pin".

Then, regardless of whether there is a selected pin, we show an "All Routes" table underneath that.

Asking for location right away is not a good experience since it puts up a generic popup asking for permission, with no opportunity to explain why we're asking.

**Comment by SeannyPhoenix** [comment]
I agree that we don't want to ask for the user's location. The routes I recommend are F, N, 9, 49.

**Comment by exxonvaldez** [comment]
How about showing the top 5 scoring lines?  Ideally with the full list of routes in the table.  Currently if we plot any routes on the map, we filter the table to the same list.  But for top 5 I don’t think we’d want to filter tge table.
